docs/testplan/Convergence measurement in data center networks.md

```diff
@@ -175,7 +177,8 @@ Effectively, on T1 routes are learned from 2 peers and forms ECMP.
 
 Fixed duration server traffic is generated for a bunch of destination prefixes, which will load balance by T1 towards its peers.<br/>
```

Comment:
can you add more details in the test the number of prefixes used in the test?

Reply:
Number of routes field will be parameterized. So, user can specify the number of routes they want to use in this test. Initially before running the test, check will be there by sending the traffic and checking no loss is observed. If the provided number of routes is not supported, it will result in packet loss while sending traffic, so that users can reduce the number of routes.